src/components/dev-dot-content/dev-dot-content.module.css

```diff
@@ -7,93 +7,88 @@
 ***
 */
 .root {
-  composes: hds-typography-body-300 from global;
+	composes: hds-typography-body-300 from global;
```

Comment:
Not sure what's going on with formatting here; I think this is changing spaces to tabs?

Reply:
How weird! That's definitely what's happening. Running prettier via the CLI command doesn't detect any issues in this file... so this must have been formatted by our lint command that runs in the pre-commit hook 🤔